odoo_sh/getting_started/branches.rst

```diff
@@ -408,13 +408,39 @@ Uploads the changes you just added in the *master* branch on your remote reposit
 
 SSH
 ---
-Connect to your builds using ssh. 
+Setup
+*****
+In order to use SSH you will need to setup your profile SSH public key (if it is not already done).
+In order to do this follow these steps :
+
+#. `Generate a new SSH key
+   <https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key>`_
+#. `Copy the SSH key to your clipboard
+   <https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account>`_
+   (only apply the step 1)
+#. Paste the copied content to your profile SSH keys and press "Add"
+
+   .. image:: ./media/SSH-key-pasting.png
```

Comment:
Your email address is showing in the image and may confuse readers as it may not be obvious to everyone that `lse@odoo.com` is the email address of an Odoo employee. I suggest hitting F12 and changing it by something like `my.email@example.com`.